docs/references/differences_with_original_sdk_markdown.md

```diff
@@ -0,0 +1,65 @@
+Differences Between the New and Old Python SDKs
+===============================================
+
+Packaging
+---------
+
+The original [Alteryx Python
+SDK](https://help.alteryx.com/current/developer-help/python-engine-sdk)
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version